setup.py

```diff
@@ -23,7 +23,7 @@ def readme():
     maintainer_email='dev' '@' 'spoqa.com',
     license='GPLv3 or later',
     py_modules=['flake8_import_order_spoqa'],
-    install_requires=['flake8-import-order >= 0.16'],
+    install_requires=['flake8-import-order <= 0.16'],
```

Comment:
Would it better to fix the version of `flake8-import-order` to `0.16`? And add some "FIXME" comment about why this library comment.

And i think it would be nice to fix flake8-import-order-spoqa works on the newer version of flake8-import-order.